pandas/io/excel/_openpyxl.py

```diff
@@ -451,3 +462,280 @@ def write_cells(self, cells, sheet_name=None, startrow=0, startcol=0,
                             xcell = wks.cell(column=col, row=row)
                             for k, v in style_kwargs.items():
                                 setattr(xcell, k, v)
+
+
+class _OpenpyxlReader(_BaseExcelReader):
+
+    def __init__(self, filepath_or_buffer):
+        """Reader using openpyxl engine.
+
+        Parameters
+        ----------
+        filepath_or_buffer : string, path object or Workbook
+            Object to be parsed.
+        """
+        err_msg = "Install openpyxl >= 2.4.0 for Excel with Openpyxl support"
+
+        try:
+            import openpyxl
```

Comment:
Just a general comment for a follow up (I don't required here) but probably better to move imports to the global space of the module rather than the init (applicable to xlrd module as well). Could do away for subclass of __init__ altogether

Reply:
I assumed this is to avoid issues when the module is not installed, would definitely be a lot clear if it doesn't cause problems

Reply:
Can you move this import check to the top level of the module?

Reply:
Feel free to ignore this request; I think we can fix this up after #26802